app/controllers/gmBulkProduct.js

```diff
@@ -279,7 +279,7 @@ module.exports = {
           }
 
           // Validate product method will returns either empty object or object with error keys
-          const validationResult = await validateProduct(product, constraints);
+          const validationResult = await validateProduct(product);
```

Comment:
Not checking Against constraints now??

Reply:
@Subbu-betalectic

Reply:
@ShreyDhyani Removed the unnecessary parameter